packages/ketcher-core/src/domain/serializers/mol/molSerializer.ts

```diff
@@ -37,27 +37,34 @@ export class MolSerializer implements Serializer<Struct> {
   deserialize(content: string): Struct {
     const molfile = new Molfile()
     const lines = content?.split(/\r\n|[\n\r]/g)
+
+    const parseCTFileParams = {
+      molfileLines: lines,
+      shouldReactionRelayout: this.options.reactionRelayout,
+      ignoreChiralFlag: this.options.ignoreChiralFlag
+    }
+
     try {
-      return molfile.parseCTFile(lines, this.options.reactionRelayout)
+      return molfile.parseCTFile(parseCTFileParams)
     } catch (ex) {
       if (this.options.badHeaderRecover) {
         try {
           // check whether there's an extra empty line on top
           // this often happens when molfile text is pasted into the dialog window
-          return molfile.parseCTFile(
-            lines.slice(1),
-            this.options.reactionRelayout
-          )
+          return molfile.parseCTFile({
+            ...parseCTFileParams,
+            molfileLines: lines.slice(1)
+          })
         } catch (ex1) {
           //
         }
         try {
           // check for a missing first line
           // this sometimes happens when pasting
-          return molfile.parseCTFile(
-            [''].concat(lines),
-            this.options.reactionRelayout
```

Comment:
this.options.reactionRelayout is not used anymore in molSerializer.ts.

I think we can delete it from class itself and from other methods like:

parseCTFile, parseRxn...